Packs/Azure-Enrichment-Remediation/Playbooks/playbook-Azure_-_User_Investigation.yml

```diff
@@ -1275,8 +1273,182 @@ tasks:
     view: |-
       {
         "position": {
-          "x": -390,
-          "y": 760
+          "x": -430,
+          "y": 920
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "37":
+    id: "37"
+    taskid: 5b2b1c42-7a11-456d-82a6-421c46feee98
+    type: regular
+    task:
+      id: 5b2b1c42-7a11-456d-82a6-421c46feee98
+      version: -1
+      name: Search for available tables
+      description: Executes an Analytics query for data.
+      script: Azure Log Analytics|||azure-log-analytics-execute-query
+      type: regular
+      iscommand: true
+      brand: Azure Log Analytics
+    nexttasks:
+      '#none#':
+      - "4"
+      - "5"
+      - "6"
+    scriptarguments:
+      extend-context:
+        simple: AzureTables=
+      query:
+        simple: search "*" | summarize count() by $table | sort by count_ desc
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -430,
+          "y": 230
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "38":
+    id: "38"
+    taskid: 6660c680-dbb2-4718-8eb2-3c52f0aa5452
+    type: condition
+    task:
+      id: 6660c680-dbb2-4718-8eb2-3c52f0aa5452
+      version: -1
+      name: should perform queries on Azure Sentinel Anomalies tables?
+      description: Checks if the Azure Sentinel Anomalies table is available for querying.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "22"
+      "yes":
+      - "13"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: containsString
+          left:
+            value:
+              simple: ${AzureTables.tables.rows.[].[0]}
```

Comment:
if all those 3 tasks are the same. lets remove the duplicated tasks. 

the context path doesnt look right. please verify it

Reply:
Each conditional task checks another table.
I checked the context path, and it is the correct path.